### ISSUE TITLE
Thermal throttling?

### ISSUE THREAD

**Issue by piegamesde** [martin31821/cpupower#62]
There is this nice script that does thermal throttling: https://github.com/Sepero/temp-throttle/

It's just a loop polling the temperature and changing the max frequency until that temperature is below a threshold.

I think it would be a nice addition to have such features and not that hard to implement. (Since it is using another interface than this extension, it might be faster to re-implement it from scratch)

**Comment by martin31821** [comment]
I don't think this is an issue, because the CPU will throttle itself on high temperatures.

**Comment by piegamesde** [comment]
Yes, but the only way this is done is through thermald and its configuration possibilities are really limited. This feature would allow thermal throttling at arbitrary temperatures. My Laptop for example throttles at 95°C, which is way too high for me. I'd prefer to setting this down to a temperature where the fan isn't needed (like ~75°C).

**Comment by victor-bayas** [comment]
> Yes, but the only way this is done is through thermald and its configuration possibilities are really limited. This feature would allow thermal throttling at arbitrary temperatures. My Laptop for example throttles at 95°C, which is way too high for me. I'd prefer to setting this down to a temperature where the fan isn't needed (like ~75°C).

I know this is an old thread but you can setup thermald config file to throttle your CPU when it reaches 75 °C

**Comment by piegamesde** [comment]
@victor-bayas That'd be awesome! I know this isn't the correct thread for this, but do you have any examples or documentation? I still can't find anything about this.

**Comment by victor-bayas** [comment]
> @victor-bayas That'd be awesome! I know this isn't the correct thread for this, but do you have any examples or documentation? I still can't find anything about this.

man thermal-conf.xml

In the example 1 change the temperature, by default it's on 86 degrees.

Put that file in etc/thermald
